thunder/torch/__init__.py

```diff
@@ -2842,6 +2842,19 @@ def _normalize(a: TensorProxy, /, norm_dims, eps: Number) -> tuple[TensorLike, T
     return out, mean, rstd
 
 
+@torchsymbol(torch.nn.functional.normalize, is_method=True)
+def normalize(
+    a: TensorProxy, /, p: float = 2, dim: int | Sequence[int] = 1, eps: float = 1e-12, out: None | TensorProxy = None
```

Comment:
nit: `p: float = 2.` so that the default for `p` is actually a float

Reply:
Oh, my bad. Thanks for catching this! 🙌